code/datums/components/food/edible.dm

```diff
@@ -373,25 +373,37 @@ Behavior that's still missing from this component that original food items had t
 	var/mob/living/carbon/human/H = M
 	if(HAS_TRAIT(H, TRAIT_AGEUSIA))
```

Comment:
Don't do this like this, revert to the old code in this and instead create a callback that can handle the checkliked behavior for donuts.

Reply:
outdated comment?

Reply:
yes